ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -45,22 +45,8 @@ tests:
     test:
     - chain: openshift-qe-cluster-density-v2
     workflow: rosa-aws-sts
-- as: perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic
+- as: perfscale-rosa-multiaz-ci-tests-24nodes-periodic
```

Comment:
Should we call this as control plane tests instead of `ci-tests`

Reply:
We have also use kube-burner in the name like [here](https://github.com/openshift/release/blob/1bd82837937cf107e83a32900496f3c61b5ccc2f/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml#L24) instead of ci-tests

Reply:
@svetsa-rh @paigerube14 :
Will update the ci-tests --> control-plane which would more clarity.

Reply:
Hi 👋 @svetsa-rh @paigerube14: 
ptal and review this PR https://github.com/openshift/release/pull/42788

Reply:
Resolving this thread as the required comments are resolved.